custom/enikshay/management/commands/run_result_summary_display_migration.py

```diff
@@ -0,0 +1,40 @@
+from custom.enikshay.management.commands.utils import BaseEnikshayCaseMigration
+
+DATAMIGRATION_CASE_PROPERTY = 'datamigation_result_summary_display'
+RESULT_SUMMARY_DISPLAY = 'result_summary_display'
+
+
+class Command(BaseEnikshayCaseMigration):
+    case_type = 'test'
+    case_properties_to_update = [
+        RESULT_SUMMARY_DISPLAY,
+    ]
+    datamigration_case_property = DATAMIGRATION_CASE_PROPERTY
+    include_public_cases = True
+    include_private_cases = False
+
+    @staticmethod
+    def get_case_property_updates(test, domain):
+        if (
+            test.get_case_property(DATAMIGRATION_CASE_PROPERTY) == 'yes'
+            or not (
+                test.get_case_property('result_recorded') == 'yes'
+                and not test.get_case_property(RESULT_SUMMARY_DISPLAY)
```

Comment:
I looked at the sample output and it doesn't appear very useful. 

I'm wondering if we should switch up the logic so that:
- For all tests where result_recorded = yes and test_type_value != ''/null
- We calculate the new result_summary_display
- If its different than the current result_summary_display, we update the test with the new result_summary_display

Reply:
updated, doing a dry run now